maintainers/maintainer-list.nix

```diff
@@ -5322,6 +5322,12 @@
     githubId = 1572058;
     name = "Leonardo Cecchi";
   };
+  leonardp = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add leonardp``.

Reply:
done